cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,229 @@ Cask Data Application Platform Release Notes
+  to update these records to the new format in the Upgrade Tool.
+
+- :cask-issue:`CDAP-9283` - The migration of ScheduleStore and AppMetadataStore from the
+  app version-less format to the new format with a version will be performed by background
+  threads running in the CDAP Master process.
+
+- :cask-issue:`CDAP-9311` - Adding support for service discovery in plugins.
+
+- :cask-issue:`CDAP-9371` - Added convenience functions to set a Date in a
+  StructuredRecord as a long or a string.
+
+- :cask-issue:`HYDRATOR-1432` - Added a timer for realtime preview to stop a run
+  automatically. By default, it is one minute.
+
+Bug Fixes
+---------
+
+- :cask-issue:`CDAP-4213` - Fixed an issue in the CDAP startup script for Windows,
+  removing the requirement of being an admin to run the CDAP process.
+
+- :cask-issue:`CDAP-5715` - Made Plugin Endpoint invocation more robust. If a plugin's
+  parent can't instantiate the plugin necessary for invoking, CDAP will attempt with other
+  parents of the plugin and try to instantiate using them before retuning error.
+
+- :cask-issue:`CDAP-6348` - Fixed a Issue in Namespace delete which caused CDAP
+  Application's test cases to fail earlier in a Windows environment.
+
+- :cask-issue:`CDAP-7138` - Fixed a problem with duplicate log lines showing for a running
+  program.
+
+- :cask-issue:`CDAP-7884` - Fixed a bug that preview cannot read from datasets in real
+  space.
+
+- :cask-issue:`CDAP-8013` - Disabled the write-to-any-dataset for sinks when in preview.
+
+- :cask-issue:`CDAP-8396` - Added a "call-to-action" when the user had successfully
+  created a new namespace.
+
+- :cask-issue:`CDAP-8668` - Modified the log viewer to only show ERROR, WARN, and INFO
+  levels of logs by default, instead of all logs as previously.
+
+- :cask-issue:`CDAP-8862` - Fix an issue with losing a few metrics when a container is
+  shutdown.
+
+- :cask-issue:`CDAP-8888` - Fixed an issue with the YARN container allocation logic so
+  that the correct container size is used.
+
+- :cask-issue:`CDAP-8913` - Improved the serializability of Tables and IndexedTables when
+  used in Spark programs.
+
+- :cask-issue:`CDAP-8918` - Fixed an issue where redeployment of an application with a
+  deleted schedule would fail.
+
+- :cask-issue:`CDAP-8945` - Moved the "add plugin" behavior from a plugin's left panel to
+  an "Add Entity" button in the CDAP Studio UI.
+
+- :cask-issue:`CDAP-8950` - Fixed an issue in the CDAP UI where navigating from a stream
+  card to an overview and then to a detail page made the detail page show a spinner icon
+  indefinitely.
+
+- :cask-issue:`CDAP-8980` - Fixed an issue with the Spark program runtime so that the Kryo
+  serializer can be used.
+
+- :cask-issue:`CDAP-9005` - Fixed an issue where the HBaseQueueDebugger failed when
+  authorization was enabled.
+
+- :cask-issue:`CDAP-9027` - Added support for grant/revoke on groups, and revoke on users,
+  in the CDAP Sentry Extension.
```

Comment:
Let's make the user the subject of a release note, not a CDAP component.

``Users can now for grant and revoke privileges for Unix groups and users when using Apache Sentry as the authorization extension for CDAP``